go/ql/lib/semmle/go/dataflow/ExternalFlow.qll

```diff
@@ -139,14 +146,14 @@ predicate sourceModel(
     row.splitAt(";", 4) = signature and
     row.splitAt(";", 5) = ext and
     row.splitAt(";", 6) = output and
-    row.splitAt(";", 7) = kind
+    exists(string k | row.splitAt(";", 7) = k and getKind(k, kind, generated))
```

Comment:
```suggestion
    getKind(row.splitAt(";", 7), kind, generated)
```

Reply:
I'm going to leave it like this so that it's the same as the other copies of `ExternalFlow.qll`. That will make it easier to update to the other copies when we get around to it.